### PR TITLE
EX-210: MSM refactor - update librtcm and gnss_convertors, add libswiftnav

### PR DESCRIPTION
**Pull request by nsirola**
Pull in https://github.com/swift-nav/gnss-converters/pull/114 and https://github.com/swift-nav/librtcm/pull/59

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.5.0-develop-2018083120-4-g6454d57b
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=657634)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.5.0-develop-2018083120-4-g6454d57b&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.5.0-develop-2018083120-4-g6454d57b&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.5.0-develop-2018083120-4-g6454d57b
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.5.0-develop-2018083120-4-g6454d57b)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.5.0-develop-2018083120-4-g6454d57b/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.5.0-develop-2018083120-4-g6454d57b

**Comment by benjaminaltieri** [comment]
Just FYI, this is very similar to a commit going into #795:

https://github.com/swift-nav/piksi_buildroot/pull/795/commits/e6a108f718cfc3ced6b5396d96b704c818a105a7

Should we pull out the commonalities here and merge as a separate PR? @anth-cole thought?

File: package/gnss_convertors/gnss_convertors.mk [hunk 1]
```diff
@@ -4,11 +4,11 @@
 #
 ################################################################################
 
-GNSS_CONVERTORS_VERSION = v0.3.56
+GNSS_CONVERTORS_VERSION = b0fc489e8956224906049e9ea7a678e38fb5c55d
```
Comment (anth-cole) [inline package/gnss_convertors/gnss_convertors.mk]: I assume this is the tip of gnss_converters? if so, can we tag another release and point at a specific release tag? Same for librtcm below.

Comment (nsirola) [reply package/gnss_convertors/gnss_convertors.mk]: This actually still points to the dev branch. After they are merged to master, will naturally tag them as releases.

**Comment by nsirola** [comment]
@benjaminaltieri I picked the `libswiftnav` changes from that PR. Whichever gets merged first, it should be a painless to rebase the other as the common parts are identical.

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.5.0-develop-2018090719-11-g4dd9fdb1
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=668636)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.5.0-develop-2018090719-11-g4dd9fdb1&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.5.0-develop-2018090719-11-g4dd9fdb1&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.5.0-develop-2018090719-11-g4dd9fdb1
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.5.0-develop-2018090719-11-g4dd9fdb1)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.5.0-develop-2018090719-11-g4dd9fdb1/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.5.0-develop-2018090719-11-g4dd9fdb1

**Comment by nsirola** [comment]
Closing as obsolete, pointers already moved by https://github.com/swift-nav/piksi_buildroot/pull/795

**Pull request closed without merging**